demos/motmetrics4norfair/motmetrics4norfair.py

```diff
@@ -120,7 +116,9 @@ def keypoints_distance(detected_pose, tracked_pose):
         distance_function=keypoints_distance,
         distance_threshold=distance_threshold,
         detection_threshold=detection_threshold,
-        point_transience=2,
+        hit_inertia_min= 10,
+        hit_inertia_max= 12,
```

Comment:
There should not be a space after the `=`. Maybe run a black formatting job to search for any more of these types of formatting issues.

Reply:
Sorry, my bad, I forgot to run a black formatting on that before pushing. Will be fixed!